qfieldsync/gui/cloud_create_project_widget.py

```diff
@@ -168,16 +171,19 @@ def cloudify_project(self):
         cloud_convertor.warning.connect(self.on_show_warning)
         cloud_convertor.total_progress_updated.connect(self.on_update_total_progressbar)
 
-        try:
-            cloud_convertor.convert()
-        except Exception:
-            QApplication.restoreOverrideCursor()
-            critical_message = self.tr(
-                "The project could not be converted into the export directory."
-            )
-            self.iface.messageBar().pushMessage(critical_message, Qgis.Critical, 0)
-            self.close()
-            return
+        def convert():
+            try:
+                cloud_convertor.convert()
+            except Exception:
+                QApplication.restoreOverrideCursor()
+                critical_message = self.tr(
+                    "The project could not be converted into the export directory."
+                )
+                self.iface.messageBar().pushMessage(critical_message, Qgis.Critical, 0)
+                self.close()
+                return
+
+        self.timer.singleShot(0, convert)
 
         self.create_cloud_project()
```

Comment:
Is it a problem that line 188 is now done before create is executed?

Reply:
If you have a fast internet - no :) Thanks for spotting it.